docs/_data/this_week.yaml

```diff
@@ -1,22 +1,22 @@
 lectures: []
 
 projects:
-- date: Fri Jan 27
-  deadline: Due Thurs Feb 2 @ 11:59pm
-  end_date: 2023-02-03 00:00:00
-  link: /projects/P2
-  name: Project 2A
-  numDays: 5
-- date: Fri Feb 03
-  deadline: Due Tues Feb 14 @ 11:59pm
-  end_date: 2023-02-15 00:00:00
-  link: /projects/P2/2_firstsprint
-  name: Project 2B
-  numDays: 8
+  - date: Fri Jan 27
+    deadline: Due Thurs Feb 2 @ 11:59pm
+    end_date: 2023-02-03 00:00:00
+    link: /projects/P2
+    name: Project 2A
+    numDays: 5
+  - date: Fri Feb 03
+    deadline: Due Tues Feb 14 @ 11:59pm
+    end_date: 2023-02-15 00:00:00
+    link: /projects/P2/2_firstsprint
+    name: Project 2B
+    numDays: 8
 
 recitation:
-  date: Mon Jan 30
```

Comment:
Could you add this same information in schedule.yaml? That populates the schedule table at the bottom of the main page; the "this_week" yaml is automatically populated by the python script

Reply:
I added the info to schedule.yaml, but still not sure how to make gradescope quiz.